queue_services/entity-filer/tests/unit/filing_processors/test_dissolution.py

```diff
@@ -37,6 +37,8 @@
     ('ULC', 'BC1234567', 'voluntary'),
     ('LLC', 'BC1234567', 'voluntary'),
     ('CP', 'CP1234567', 'voluntary'),
+    ('BC', 'BC1234567', 'administrative'),
+    ('SP', 'FM1234567', 'administrative'),
```

Comment:
can we rename the test name from `test_voluntary_dissolution` to `test_dissolution`